zerver/webhooks/flock/doc.md

```diff
@@ -0,0 +1,32 @@
+# Flock webhook
+
+Get Zulip notifications from your Flock channels.
+
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+Login into to your [Flock](https://flock.com) account, follow below
+steps given below to get started.
+
+1. Go to **Apps** in the right bottom.
+
+![](/static/images/integrations/flock/001.png)
+
+2. Then, Go to **Admin Panel** -> **Webhooks**.
+
+3. Click on **Add** button next to **Outgoing Webhook**.
+
+![](/static/images/integrations/flock/002.png)
+
+4. Enter the URL you want webhooks to be sent to, in **Callback URL** field.
+
+![](/static/images/integrations/flock/003.png)
+
+5. After filling in other details, click **Save Settings**.
```

Comment:
I'd recommend elaborating on this step. What are the "other details", are they required?